.github/workflows/km-ci-workflow.yaml

```diff
@@ -553,6 +549,21 @@ jobs:
         run: make -C payloads test-all-withdocker HYPERVISOR_DEVICE=/dev/kkm DOCKER_INTERACTIVE=
         timeout-minutes: 20
 
+      # get static krun created by krun-static-build job
+      - uses: actions/download-artifact@v2
+        with:
+          name: krun-static
+          path: build/container-runtime
+
+      - name: rename krun.static to krun
```

Comment:
name isn't right

Reply:
I went over the whole file to make sure names make sense

Reply:
All names are now present (no unnamed steps ) and consistent, I hope